qulice-pmd/src/test/java/com/qulice/pmd/PMDValidatorTest.java

```diff
@@ -491,4 +497,19 @@ public void allowsAssertFail()
         ).validate();
     }
 
+    /**
+     * PMDValidator can allow non-static, non-transient fields.
+     * @throws Exception If something wrong happens inside.
+     */
+    @Test
+    public void allowNonTransientFields() throws Exception {
```

Comment:
@krzyk I am confused. From one side we have policy of documentation like `Class can do something` and I received recommendations for improvement in [previous ticket](https://github.com/teamed/qulice/pull/665#discussion-diff-51991973). 
Or `PMDValidator can allow`
Or `PMDValidator allows` 
What is correct?